flow-tests/test-frontend/pom.xml

```
@@ -12,6 +12,11 @@
     <packaging>pom</packaging>
 
     <modules>
+        <!-- shared assets -->
+        <module>embedding-test-assets</module>
+        <module>addon-with-templates</module>
```

```suggestion
        <module>addon-with-templates</module>
        <module>vite-test-assets</module>
```
Copy paste name and wrong order.

---

😣 , done